src/wiki/views/article.py

```diff
@@ -15,6 +15,7 @@
 from django.views.generic.base import RedirectView, TemplateView, View
 from django.views.generic.edit import FormView
 from django.views.generic.list import ListView
+from django.views.decorators.clickjacking import xframe_options_sameorigin
```

Comment:
The seemingly non-alphabetic location of this import, made me wonder if 1) you have done `pip install pre-commit`  + `pre-commit install` and 2) if linting is run properly on Travis.. will look into that..

Reply:
Linting on Travis is back in `master`